dbusmock/templates/bluez5.py

```diff
@@ -253,6 +259,47 @@ def AddAdapter(self, device_name, system_name):
         ],
     )
 
+    bluez = mockobject.objects["/org/bluez"]
+
+    # Advertising Manager
+    if bluez.enable_advertise_api:
+        max_advertisement_instances = 5
+        advertising_manager_properties = {
+            "ActiveInstances": dbus.Byte(0, variant_level=1),
+            "SupportedInstances": dbus.Byte(max_advertisement_instances, variant_level=1),
+            "SupportedIncludes": dbus.Array(["tx-power", "appearance", "local-name", "rssi"], variant_level=1),
+            "SupportedSecondaryChannels": dbus.Array(["1M", "2M", "Coded"], variant_level=1),
+            "SupportedCapabilities": dbus.Dictionary(
+                {
+                    "MaxAdvLen": dbus.Byte(251),
+                    "MaxScnRspLen": dbus.Byte(251),
+                    "MinTxPower": dbus.Int16(-34),
+                    "MaxTxPower": dbus.Int16(7),
+                },
+                signature="sv",
+                variant_level=1,
+            ),
+            "SupportedFeatures": dbus.Array(
+                [
+                    "CanSetTxPower",
+                    "HardwareOffload",
+                ],
+                variant_level=1,
+            ),
+        }
+        adapter.AddProperties(LE_ADVERTISING_MANAGER_IFACE, advertising_manager_properties)
+        adapter.AddMethods(
+            LE_ADVERTISING_MANAGER_IFACE,
+            [
+                ("RegisterAdvertisement", "oa{sv}", "", RegisterAdvertisement),
+                ("UnregisterAdvertisement", "o", "", UnregisterAdvertisement),
+            ],
+        )
+
+        # Track advertisements per adapter
+        adapter.max_advertisement_instances = max_advertisement_instances
```

Comment:
`max_advertisement_instances` never gets changed -- do you plan to do this in the future? Otherwise this is just always going to be 5. I.e. it could just as well be a global constant.

Reply:
Yes, good point. Was vaguely thinking about making this configurable with a parameter to the template, but then didn't see a compelling use case, so a constant would do just fine.

Reply:
Made this a constant instead.